Enterprise-CMCS/macpro-mdct-mcr
# Use spies for console output in tests

### Description

- Use spies for tests that intentionally output console messages

### Related ticket(s)


CMDCT-3835

---
### How to test

1. cd to `macpro-mdct-mcr/services/app-api`
2. Run `yarn coverage`
3. Verify tests pass and coverage is unaffected
4. Verify console messages are gone






### Pre-review checklist
